lessons/13-server-rendering/README.md

```diff
@@ -50,7 +46,7 @@ module.exports = {
 
   module: {
     loaders: [
-      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader?presets[]=es2015&presets[]=react' }
+      { test: /\.js$/, loader: 'babel-loader?presets[]=es2015&presets[]=react' }
```

Comment:
This Webpack config is only used for the server where we set all the node modules to be external dependencies. This means Webpack does not include them in the bundle and does not apply loaders such as Babel to them. Therefore, the property "is unnecessary as node_modules are externalized and thus not bundled by webpack".